### PR TITLE
Require opt in

### PR DESCRIPTION
**Pull request by alexjch**
Change to use explicit opt-in and documentation update to note the change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,38 +67,41 @@ Descriptions of config options are listed below in the Usage section.
 
 Starting the client
 ---------------------
+To use the telemetrics cient a one time explicit ```opt-in``` is required (this is
```
Comment (jurobystricky) [inline README.md]: cient -> client

**Comment by jurobystricky** [comment]
There are also man pages that refer to opt-out file. These should be updated as well.

**Comment by tpleavitt** [comment]
@alexjch , do you know when the PR will be implemented?

**Comment by alexjch** [comment]
@tpleavitt assuming everything is good it will take 2 to 3 weeks.

**Pull request merged**